mock/docs/site-defaults.cfg

```diff
@@ -463,6 +463,16 @@
 
 ## Use this to force foreing architecture (requires qemu-user-static)
 # config_opts['forcearch'] = None
+## mapping from target_arch (or forcearch) to arch in /usr/bin/qemu-*-static
+# config_opts['qemu_user_static_mapping'] = {                                                                                                                                              
+#     'aarch64': 'arm',
```

Comment:
qemu-user-static-aarch64 seems avaliable

Reply:
ahh! I did not seen it because I removed it on my system do test error msg. Will change.